src/lib/entity-state.ts

```diff
@@ -69,22 +70,7 @@ export abstract class EntityState<T> {
     this.storePath = storeClass['NGXS_META'].path;
```

Comment:
'NGXS_META' - put in a constant somewhere

Reply:
Added 

```typescript
export const NGXS_META_KEY = 'NGXS_META';
```